testproject/Assets/Tests/Runtime/MultiprocessRuntime/BaseMultiprocessTests.cs

```diff
@@ -22,15 +23,12 @@ public abstract class BaseMultiprocessTests
     {
         private string m_LogPath;
         private bool m_HasSceneLoaded = false;
-        // TODO: Remove UTR check once we have Multiprocess tests fully working
-        protected bool IgnoreMultiprocessTests => MultiprocessOrchestration.ShouldIgnoreUTRTests();
+        protected virtual bool LaunchRemotely => false;
 
         protected virtual bool IsPerformanceTest => true;
```

Comment:
Any reason for separate LaunchRemotely and IsPerformanceTest? We could just have LaunchRemotely and replace perf test checks for remote checks?

Reply:
Performance tests still require a separate setup and infrastructure that I still haven't built. We could just use the same multi-machine tooling for them but I hadn't tested any of that thus far and so kept it separate. @SamuelBellomo  is that okay?

Reply:
yeah you're right, until we know for sure whether there's going to be differences or not, it's wiser to keep them separate.
Just something to keep in mind when we do address these.